.github/workflows/ci.yml

```diff
@@ -1,5 +1,13 @@
 name: Continuous Integration
-on: [push, pull_request]
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
I am not sure if this is really necessary.

Reply:
Probably not, we do however do this in many of our repos, and the current workflow also runs for tags.